lib/livebook_web/live/home_live/session_list_component.ex

```diff
@@ -171,6 +185,42 @@ defmodule LivebookWeb.HomeLive.SessionListComponent do
     """
   end
 
+  defp edit_sessions(assigns) do
+    ~H"""
+    <div class="mx-5 text-gray-600 flex flex-row gap-1">
+      <%= if @editing? do %>
+        <button class="button-base button-outlined-gray px-4 pl-0 py-1" phx-click="cancel">
+          <.remix_icon icon="close-line" class="text-lg leading-none align-middle ml-1" />
+          <span>Cancel</span>
+        </button>
+        <.menu id="edit-sessions">
+          <:toggle>
+            <button class="button-base button-outlined-gray px-4 py-1">
+              <span>Actions</span>
+              <.remix_icon icon="arrow-down-s-line" class="text-lg leading-none align-middle ml-1" />
+            </button>
+          </:toggle>
+          <:content>
+            <%= for action <- ["close_all", "disconnect"] do %>
+              <%= live_patch to: Routes.home_path(@socket, :edit_sessions, action),
+              class: "menu-item #{if action == "close_all", do: "text-red-600", else: "text-gray-600"}",
+              role: "menuitem" do %>
+              <.remix_icon icon={action_icon(action)} />
+                <span class="font-medium"><%= action_label(action) %></span>
+              <% end %>
+            <% end %>
+          </:content>
+        </.menu>
+      <% else %>
+        <button class="button-base button-outlined-gray px-4 pl-0 py-1" phx-click="editing?">
+          <.remix_icon icon="edit-box-line" class="text-lg leading-none align-middle ml-1" />
+          <span> Edit sessions </span>
```

Comment:
If the icon makes the operation clear, I think we could put the text into a tooltip to keep it less verbose :)